### ISSUE TITLE
export_quant() does not handle the merged tidyproteomics data object

### ISSUE THREAD

**Issue by tywang-tw** [jeffsocal/tidyproteomics#21]
Hi Jeff, The export_quant() function seems not work for the merged object when quantitative_source = 'selected' in merge(). 
Here is the error I got:
Error in `extract()`:
! `values` must be one of "selected", not "raw".

**Comment by jeffsocal** [comment]
Thanks for pointing this out. I have addressed the underlying issue and pushed out v1.8.4. I should note that it is unadvised to normalize two sets independently then merge, which results in the `selected` moniker for the quantitative value - meaning that the selected normalized quant values have been merged. There are some possible corner cases where this is appropriate. However, for the vast majority of analyses, it is recommended to merge using the `raw` quant values, the default behavior, then normalize. Normalizing prior to merging, as the possibility then exists to have different normalization methods used between sets and major quantitative offsets, that normalization is intended to address, persist or become exaggerated, increasing the probability of false positives in downstream analyses.
